client/components/command-palette/use-command-palette.tsx

```diff
@@ -73,9 +84,17 @@ interface SiteToActionParameters {
 	};
 }
 
+function isCustomDomain( siteSlug: string | null | undefined ): boolean {
+	if ( ! siteSlug ) {
+		return false;
+	}
+	return ! siteSlug.endsWith( '.wordpress.com' ) && ! siteSlug.endsWith( '.wpcomstaging.com' );
+}
+
 const useSiteToAction = () => {
-	const dispatch = useDispatch();
-	const currentRoute = useSelector( ( state: object ) => getCurrentRoutePattern( state ) );
+	// TODO: Find an alternative way to use the current route.
```

Comment:
A route in the calypso sense doesn't make sense in a wp-admin context. It might be better instead to use something like window.location.pathname, replacing the site in the path domain with ':site' 🤔